test/unit/Dockerfile.nodejs20.x

```diff
@@ -0,0 +1,7 @@
+FROM node:20
+RUN apt-get update
```

Comment:
I could have simplified into one single Dockerfile and passing the NODEJS_MAJOR as build arg, but I feel it's short enough to have it 3 times, allowing us to have potential drift in Dockerfile later